src/wasm-binary.h

```diff
@@ -261,17 +261,32 @@ enum ASTNodes {
   F64Le = 0x9a,
   F64Gt = 0x9b,
   F64Ge = 0x9c,
```

Comment:
Just I don't personally like having tables with symbols that differ from the spec and reordering them in the process. It is a lot of extra work for the read, it was a lot to scrutinize.